src/main/java/org/spongepowered/api/world/gen/Populator.java

```diff
@@ -61,59 +61,15 @@
-     * {@code (chunkX + 1, chunkZ + 1)} will at least have been generated
-     * (although they do not have to be populated) to allow
-     * {@link PopulatorObject}s to overlap across chunk boundaries.
+     * Applies the populator to the given {@link Extent} volume. The entire area
+     * of the given extent should be populated.
      *
-     * <p>This means that there are only four chunks guaranteed to be loaded.
-     * Other chunks in the world may or may not be loaded. Avoid touching those
-     * chunks, trying to get/set a block there may cause the chunk to be loaded,
-     * which is a bad thing during terrain population.</p>
-     *
-     * <p>Those four chunks form an area of 32x32 columns that you can populate,
-     * from {@code (block x, block z) (chunkX * 16, chunkZ * 16)} to
-     * {@code (chunkX * 16 + 31, chunkZ * 16 + 31)}. See <b>Figure 1</b>. To
-     * effectively use this area, it is recommend to make sure the centers of
-     * all objects are placed from
-     * {@code (block x, block z) (chunkX * 16 + 8, chunkZ * 16 + 8)} to
-     * {@code (chunkX * 16 + 23, chunkZ * 16 + 23)}. This ensures both that each
-     * object can extend 8 blocks on the x and z axis from its center and that
-     * every area in the world is populated exactly once.</p>
-     *
-     * <pre>
-     *       +----------+----------+ . . The chunk provided as a parameter
-     *       |          |          | . . to this method.
-     *       |          |          |
-     *       |     #####|#####     | ### The area you should populate.
-     *       |     #####|#####     | ###
-     *       +----------+----------+
-     *       | . . #####|#####     |
-     *       | . . #####|#####     |
-     *       | . . . . .|          |
-     *       | . . . . .|          |
-     *       +----------+----------+
-     * </pre>
-     * 
-     * <p><b>Figure 1</b> <i>The four chunks guanteed to be loaded, along with
-     * the area you are allowed to populate.</i></p>
-     *
-     * <p>The following code guantantees that the center of your object is
-     * placed with its center in the recommend area:
-     * 
-     * <pre>
-     *  {@link Vector3i} chunkStartBlockPos = chunk.getPosition().mul(16);
-     *  Vector3i populationAreaStartBlockPos = chunkStartBlockPos.add(8, 0, 8);
-     *  Vector3i objectCenterBlockPos = populationAreaStart.add(
-     *          random.nextInt(16), justSomeValueForY, random.nextInt(16));
-     * </pre>
-     *
-     * @param chunk The provided chunk.
+     * @param volume The volume to be populated
+     * @param world The World within which the generation in happening
      * @param random A random number generator. This random number generator is
      *        based on the world seed and the chunk position. It is shared with
      *        with other populators.
      */
-    void populate(Chunk chunk, Random random);
+    void populate(Extent volume, World world, Random random);
```

Comment:
It seems reversed like this - the `Extent` is part of the `World`, this makes it sound the opposite.